test_app/after/src/indent_18.erl

```diff
@@ -17,8 +17,9 @@
           they:also(should:be(indented_using:break_indent(1)))}).
 
 infix_expr() ->
- this:infix(expression) ++
-  should:be(indented) ++ using:break_indent(1).
+ this:infix(expression)
+ ++ should:be(indented)
```

Comment:
I like this, but I think I'd indent it. In any case it's better than it was, since, at a glance, I'm not inclined to think `should:be` is a "side-effect" function, but a continuation over something that came before.

Reply:
Well... The idea is to format all I fix operators in the same way 🤷‍♂️